crates/chain/src/tx_graph.rs

```diff
@@ -715,6 +717,30 @@ impl<A: Anchor> TxGraph<A> {
         changeset
     }
 
+    /// Inserts the given `missing_at` for `txid`
+    pub fn insert_missing_at(&mut self, txid: Txid, missing_at: u64) -> ChangeSet<A> {
```

Comment:
`insert_evicted_at`?

```suggestion
    pub fn insert_evicted_at(&mut self, txid: Txid, evicted_at: u64) -> ChangeSet<A> {
```